Makefile

```diff
@@ -86,23 +87,26 @@ endif
 
 checks: go-version gofmt-src golint-src govet-src misspell-src
 
-run-build: deps checks clean
+compile:
 	cd $(GOPATH)/src/github.com/contiv/netplugin && \
 	NIGHTLY_RELEASE=${NIGHTLY_RELEASE} BUILD_VERSION=${BUILD_VERSION} \
 	TO_BUILD="${TO_BUILD}" VERSION_FILE=${VERSION_FILE} \
 	scripts/build.sh
 
+# fully prepares code for pushing to branch, includes building binaries
+run-build: deps checks clean compile
+
+compile-with-docker:
```

Comment:
Could you set up these tasks like this, please?
`host-compile-with-docker:` -what we have in this target
`compile-with-docker:` - a wrapper which builds this in the Vagrant VM (e.g.: `vagrant ssh netplugin-node1 -c 'bash -lc "source /etc/profile.d/envvar.sh && cd /opt/gopath/src/github.com/contiv/netplugin && make host-compile-with-docker"'`)

It should be possible to invoke these targets on the host and on the VMs.